.github/workflows/haskell.yaml

```diff
@@ -0,0 +1,67 @@
+name: Haskell
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+    paths:
+      - "haskell/**"
```

Comment:
Is it possible to also add `.github/workflows/haskell.yaml` to this list, so that modifications to the workflow trigger it ?  which also helps check that it is actually working on this PR

Reply:
Great idea, added it to all them